app/workers/articles/dev_feeds_import_worker.rb

```diff
@@ -0,0 +1,17 @@
+# NOTE: [rhymes]
+# This script will soon be removed. We need it to collect monitoring data on
+# Datadog about the production behavior of the `Feeds::Import` class
+module Articles
+  class DevFeedsImportWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :medium_priority, retry: 10
+
+    def perform
+      return unless SiteConfig.community_name == "DEV"
```

Comment:
nit: We have a handy `SiteConfig.dev.to?` that would it easier to grep in the future.

Reply:
ahha right! this worker is temporary though :D